### DISCUSSION TITLE
I hope the token spoofing gets patched.

### DISCUSSION THREAD

**Discussion by snowleopardd** [YimMenu/YimMenu#3235] [General]
Most bots nowadays will just join, take host and start spamming, you cant even kick them, it's so sad

**Comment by maybegreat48** [comment]
What region are you in?

**Reply by snowleopardd** [reply]
South america

**Comment by xesdoog** [comment]
Playing GTA Online in public sessions is like raw dogging a random crackhead in a dark alley. maybegreat has been doing a great job of tracking and blocking them so if you have any useful information it would definitely help.
